docs/RelayAPI.md

```diff
@@ -825,6 +826,56 @@ curl --header "Content-Type: application/json" \
 
 ---
 
+### Fees for meta transaction
+Get the fees the user should pay for a given meta transaction to be relayed.
+#### Request
+```
+POST /meta-transaction-fees
+```
+#### Data Parameters
+| Name            | Type                   | Required | Description            |
+|-----------------|------------------------|----------|------------------------|
+| metaTransaction | object                 | YES      | MetaTransaction object |
+
+
+The MetaTransaction object must have the following fields:
+
+| Name                  | Type     | JSON Type                                 | Description |
+|-----------------------|--------- |-------------------------------------------|-------------|
+| from                  | address  | string - hex-encoded prefixed with "0x"   | address of identity contract |
+| to                    | address  | string - hex-encoded prefixed with "0x"   | the address on which the call of the meta transaction is happening |
+| value                 |BigInteger| string                                    | the amount of wei to be sent along from 'from' to 'to'             |
+| data                  | bytes    | string - hex-encoded prefixed with "0x"   | the data object encoding the function call including arguments     |
+| delegationFees        |BigInteger| string                                    | the fees the delegate will receive for the meta transaction        |
```

Comment:
Why do we send those two values if they do not matter? As I understood it we ask the delegate where and how much he would take to relay the transaction. Is it ever planned to include some logic that evaluates the "suggestion" of the user?

Reply:
It could be that the user wants to suggest some currency network for the fees (based on current stupid implementation, he actually should). For the initial delegationFees I cannot think of a reason to have them for now but if the user does not provide them, a default of `0` will be used: https://github.com/trustlines-protocol/relay/pull/353/files?file-filters%5B%5D=#diff-afdabd145e6edf8eefd3fc20ff77ce93R40

Reply:
(There is also a default for currency network of fees)

Reply:
Okay, can we mark them as optional and tell the default?

Reply:
You mean in the docs?

Reply:
Yes. I would like to know that I must come with a suggestion and still can get the delegates preferred fee options.